demos/funnyThingsApp/composeHead.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+x-yarp-base: &yarp-base
+  image: icubteamcode/superbuild-icubhead:master-unstable_binaries
+  environment:
+    - YARP_FORWARD_LOG_ENABLE=1
+    - YARP_ROBOT_NAME
+  volumes:
+    - "${HOME}/${YARP_CONF_PATH}:/root/.config/yarp"
+    - "/dev:/dev"
+  ports:
+    - "10000:10000"
+  network_mode: "host"
+  privileged: true
+
+services:
+  yFunnyThingsApp:
+    <<: *yarp-base
+    command: ls -la	
```

Comment:
Do we need this file? it is not doing anything in the deployment as it is

Reply:
no we do not